lib/node_modules/@stdlib/complex/parse-float64/README.md

```diff
@@ -0,0 +1,142 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# parseComplex128
+
+> Parse a string representation of a 128-bit [complex number][@stdlib/complex/float64].
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var parseComplex128 = require( '@stdlib/complex/parse-float64' );
+```
+
+#### parseComplex128( str )
+
+Parses a string representation of a 128-bit [complex number][@stdlib/complex/float64]. Throws an error if the string is not recognized as a complex number.
+
+
+```javascript
+var str = '5 + 3j';
+
+try {
+    var z = parseComplex128( str );
+    // returns <Complex128>
+} catch (error) {
+    console.error('An error occurred:', error.message);
+}
+```
+
+For details on the string format, see [Complex128][@stdlib/complex/float64].
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var Complex128 = require( '@stdlib/complex/float64' );
+var parseComplex128 = require( '@stdlib/complex/parse-float64' );
+
+var str = '5 + 3j';
+try {
+    var z = parseComplex128( str );
+    var bool = ( z instanceof Complex128 );
+    // returns true
+
+    bool = ( z.re === 5.0 );
+    // returns true
+
+    bool = ( z.im === 3.0 );
+    // returns true
+} catch (error) {
+    console.error('An error occurred:', error.message);
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- Section to include cited references. If references are included, add a horizontal rule *before* the section. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="references">
+
+</section>
+
+<!-- /.references -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
+
+## See Also
+
+-   <span class="package-name">[`@stdlib/complex/float64`][@stdlib/complex/float64]</span><span class="delimiter">: </span><span class="description">128-bit complex number.</span>
+-   <span class="package-name">[`@stdlib/complex/parse-float32`][@stdlib/complex/parse-float32]</span><span class="delimiter">: </span><span class="description">parse a string representation of a 64-bit complex number.</span>
```

Comment:
```suggestion
<section class="related">
```

As mentioned in L113, we use automation for populating the related packages section.